includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2227,4 +2229,30 @@ public function get_upe_available_payment_methods() {
 			]
 		);
 	}
+
+	/**
+	 * Enable UPE feature flag. CTA from Admin Notes. See WC_Payments_Notes_Additional_Payment_Methods.
+	 */
+	public function maybe_enable_upe_feature_flag() {
+		$nonce = '';
+		if ( ! empty( $_GET['_wpnonce'] ) ) {
+			$nonce = sanitize_key( $_GET['_wpnonce'] );
+		}
+
+		if (
+			! wp_verify_nonce( $nonce, 'enable-upe' ) ||
+			empty( $_GET['page'] ) ||
+			'wc-settings' !== $_GET['page'] ||
+			empty( $_GET['action'] ) ||
+			'enable-upe' !== $_GET['action']
+		) {
+			return;
+		}
+
+		// Enable UPE.
+		WC_Payments_Features::update_upe_enabled( true );
```

Comment:
The idea of this function was to encapsulate the `_wcpay_feature_upe` flag name inside the feature class, following the approach of the getters inside that class. However, the latest `develop` introduces a `WC_Payments_Features::UPE_FLAG_NAME` const so this would suffice. Also agree on removing wrapper function that has no logic inside. 

> after enabling the feature flag, should we delete the note? 

I think it makes sense to.